### ISSUE TITLE
CUSA03173 Bloodborne: Dark spots in the DLC only happening on Intel CPUs

### ISSUE THREAD

**Issue by Missake212** [shadps4-emu/shadPS4#1594]
While looking around in the DLC, I noticed what I would qualify as weird black/dark spots, at first glance I could tell it wasn't this way on real hardware. Here's how it looks on my game:
![Capture d'écran 2024-11-19 055744](https://github.com/user-attachments/assets/aeb9345a-3e7c-47ee-80c0-e10bf8c27029)

And here's how it looks like on real hardware (couldn't get a screenshot since I can't go online on my ps4):
![GcubCF1WoAAJCFS](https://github.com/user-attachments/assets/8bcafb3d-49c5-4771-b01a-68a89ed1cea8)

Intrigued, I asked around in the BB channel if people were willing to go there and check for themselves, and I quickly noticed a pattern. My rig has an Intel cpu (13900k) and an Nvidia gpu (4090), I asked brad from the discord if he could test and here's how it looks for him (he has an AMD 5600x and an Nvidia 2080 Super):
![zzzzz](https://github.com/user-attachments/assets/a0ffddf8-e3eb-4b29-b4b8-43261170a883)

Seeing this we can tell that it appears to render normally on systems with AMD cpus. I asked for some more testing and here are the results I got from the Discord, black means the textures appear black and dark, normal means it renders like the game on original hardware:
![Capture d'écran 2024-11-25 101408](https://github.com/user-attachments/assets/6a63bcdb-f36b-4655-9036-85573e9ad5d4)

I don't know why exactly this issue appears on system with Intel cpus, also it's not like the sfx issue where it affects only 12th, 13th and 14th gen, it also affects older intel cpus. I want to add that I've tested multiple different builds, some with tess, some without, and the issue always happened for me, so I'm pretty sure it's a hardware issue. I noticed more spots in the DLC where this issue happens, though I didn't go further than Ludwig myself, so there's maybe even more, and I also didn't really check the base game to see if this also happens outside of the DLC. Here are a few more examples of those dark textures:
![Capture d'écran 2024-11-23 010015](https://github.com/user-attachments/assets/fe96cf51-5830-4d11-bd1b-5b9be46d3b8f)
![Capture d'écran 2024-11-23 010023](https://github.com/user-attachments/assets/136517a8-271f-40f9-bbb6-b6d2a01f8494)
![Capture d'écran 2024-11-23 010031](https://github.com/user-attachments/assets/b1714f00-84df-4049-92c4-2e03a894b32f)
![Capture d'écran 2024-11-24 174114](https://github.com/user-attachments/assets/84f7979d-4b32-4a7e-969d-590678dfdef3)
![Capture d'écran 2024-11-24 174448](https://github.com/user-attachments/assets/6c32b26a-d6f3-4e55-8b32-981e85f121ce)
![Capture d'écran 2024-11-24 174518](https://github.com/user-attachments/assets/d568febb-fc13-4b35-b416-79af44af2f07)
![Capture d'écran 2024-11-24 174541](https://github.com/user-attachments/assets/3f517af4-3e28-4aa0-a050-1ecf087949d6)

Finally, following brad's suggestion, I tried to see if removing baked lights would maybe have an effect on the issue, and it seems he was right. Here's how it looks like with the baked lights removed, it sometimes kind of solve the issue, but some other times make everything black:
![Capture d'écran 2024-11-24 202727](https://github.com/user-attachments/assets/e7c499c2-bcdc-46c1-8c5b-ed0ea2e5a375)
![Capture d'écran 2024-11-24 202756](https://github.com/user-attachments/assets/a0ee134f-f4ee-47db-afd0-48c4a94d377b)
![Capture d'écran 2024-11-24 202619](https://github.com/user-attachments/assets/112cab7f-5815-4db4-b403-925bb1c40b42)
![Capture d'écran 2024-11-24 202636](https://github.com/user-attachments/assets/990591a9-9336-46d0-a5b8-3e0e4b2053ab)
![Capture d'écran 2024-11-24 202557](https://github.com/user-attachments/assets/67378e87-10d4-4efa-8d75-3d948be1daa6)

Also, shining a light source, such as the player's light or a torch for example seems to make the darkened texture display a little bit closer to real hardware:
![Capture d'écran 2024-11-23 010306](https://github.com/user-attachments/assets/b86f4758-f543-4317-b193-c2facbd9b62f)
![Capture d'écran 2024-11-23 010303](https://github.com/user-attachments/assets/fdd818c0-90f6-4899-a1bd-2b0c7a72c73e)
![Capture d'écran 2024-11-24 174114](https://github.com/user-attachments/assets/f88b3c35-367e-4739-a9c9-debc52b8ba3e)
![Capture d'écran 2024-11-24 174151](https://github.com/user-attachments/assets/c8cd003f-6fa6-456e-9b62-cd039725244d)

Here are a few more screenshots from brad (with his amd cpu):
![qzdzqdzqdzqqzddzq](https://github.com/user-attachments/assets/861d37d4-5dc7-4b2a-93ce-7d8fe8d3d2f3)
![gdfgdfgfdg](https://github.com/user-attachments/assets/c9a74d19-837c-48b0-9819-ff4c57583457)
![hjkjhkjhkjhkjh](https://github.com/user-attachments/assets/fc6630d4-1f08-43e0-a032-3811ad4d6d99)

If you find any more spots, whether in the DLC or in the base game, feel free to share your screenshots here, don't know if there's much the devs can do about this, and I'm sure there's many other issues that should be taken care of first before this one, but I wanted to post about it anyway, hoping that more people are aware of it and don't open a new issue with barely any valuable information, or ask the devs and contributors in the help channel about it.

**Comment by GHU7924** [comment]
Yes, there is such a problem.
![1](https://github.com/user-attachments/assets/4817266b-3286-471b-973f-b60771b93e01)
![2](https://github.com/user-attachments/assets/af2a4e7d-6ff5-470a-a5c4-263f7453434b)

It turns out something like there is not enough light source...
The hair of dogs is also.

![3](https://github.com/user-attachments/assets/9369b86f-ceb5-45e1-9edc-3c95c68b9898)
![4](https://github.com/user-attachments/assets/8f91ab18-c7c5-4e29-b2c2-04c1220e3889)
![5](https://github.com/user-attachments/assets/38f0a571-8686-4441-a7e9-5d487fb9a609)

I also have some kind of ripples on the textures or whatever it is, I don't know what to call it correctly.


https://github.com/user-attachments/assets/7b0cf62d-8dde-44e4-984c-4ef3430449d3

**Comment by Missake212** [comment]
Yeah I noticed those two issues (the dog's hair and the flickering on the ground) but I'm not sure if all these issues are related or not.

**Comment by GHU7924** [comment]
As a result, I still did not understand what was broken, either the lighting or the shadows. Here is an example, compare the shadow of the main character and the shadow of the tombstone.

![1234](https://github.com/user-attachments/assets/a46d53a0-f9cd-4e2c-bf7e-cfb3560c173a)

Of course, the problem may be something else.

[shad_log.txt](https://github.com/user-attachments/files/17959950/shad_log.txt)

Patches:
Skip Intro
Disable Chromatic Aberration
Disable Motion Blur
Disable HTTP Requests
Force Enable Old Hunters DLC

the situation is the same without patches.

Edit:

At first I didn't notice that there are different textures in this place, one of them seems to be covered with something.

![1](https://github.com/user-attachments/assets/dc08058e-b88d-47c8-bc70-71672c9b34c3)
![2](https://github.com/user-attachments/assets/74b7c326-c02c-4c23-9c1c-e09138fc263a)
![3](https://github.com/user-attachments/assets/346d4a27-893f-4728-a7e0-2fc34f5521a0)

The problem may be something else.

**Comment by Foul-Tarnished** [comment]
Try #1451 ?

Pretty outdated but the fix may help

**Comment by Missake212** [comment]
> Try #1451 ?
> 
> Pretty outdated but the fix may help

Any way to download an expired build? Tried to search how a little bit but couldn't find a concrete answer, I'm new to Github

**Comment by GHU7924** [comment]
Because of the many builds in different directions, you start to get confused about when and which build improved something for you or, conversely, worsened it. 
I launched the DLC not so long ago and I don't remember if there were such problems, so to check it out, I started downloading different builds, including version 0.4.0 and even version 0.3.0. It's the same there. So I thought maybe it's not about the emulator.
With the exception of various builds and their updates, the only thing that has changed on my PC lately is updating Nvidia drivers (I now have driver 566.14 installed on November 12) and reinstalling Windows 11 from version 23H2 to version 24H2.

I wanted to try checking 3 driver versions: 566.03 (October 22nd), then 565.90 (October 1st) and then 561.09 (September 11th). If none of these drivers fixes anything, it means that the problem is in something else, it is quite possible that another error occurred in Windows 11 Pro 24H2 or something else is the matter. But I'm postponing the check for now.

**Comment by Missake212** [comment]
> Try #1451 ?
> 
> Pretty outdated but the fix may help

Found a way to try the branch, as you can see it doesn't fix it at all, it's something else.
![Capture d'écran 2024-12-03 093416](https://github.com/user-attachments/assets/daede398-bcb8-487c-aed4-5b96624299e8)

**Comment by Felps3000** [comment]
I noticed some differences when using the mod to remove baked lighting.

Builds tested:
[main c019b54](https://github.com/shadps4-emu/shadPS4/actions/runs/12165907580)
[diegolix mainBB 3d5aaca](https://github.com/diegolix29/shadPS4/actions/runs/12176063426)

**Hardware:**
_i5 10400f
RTX 3060Ti
16GB 2666MT/s_

**Patches:** 
_Skip Intro
30 FPS Fix (Proper Frame Pacing)
Disable HTTP Requests_

**Mods:**
_Xbox 360 Buttons Prompt
Vertex Explosion Fix
Remove Baked Lighting (when indicated)_

**With baked lighting:**
![2024-12-05](https://github.com/user-attachments/assets/ff665476-bcf8-4918-965b-df9caf4a4f5c)

**Without baked lighting:**
![2024-12-05 (3)](https://github.com/user-attachments/assets/8c1a7c89-a17b-4b30-bb3e-2ee7d40e46f4)

**With baked lighting:**
![2024-12-05 (1)](https://github.com/user-attachments/assets/7f8eb349-236b-4eec-b281-8d74319728a4)

**Without baked lighting:**
![2024-12-05 (4)](https://github.com/user-attachments/assets/8d2f85b6-04ef-4656-be02-cd2e0d0cf290)

**With baked lighting:**
![2024-12-05 (2)](https://github.com/user-attachments/assets/a4e4d003-d477-4218-85c2-ef1d74847a17)

**Without baked lighting:**
![2024-12-05 (5)](https://github.com/user-attachments/assets/0dec2e50-82b6-4b83-ba9a-d8dc08cf8095)

**Comment by NimboYT** [comment]
I'm facing the same issue on my i9 9900k & 4070, is there no solution yet??

**Comment by Missake212** [comment]
> I'm facing the same issue on my i9 9900k & 4070, is there no solution yet??

Nope, no fixes, this is just a visual issue tbh not that big of a deal.

**Comment by NimboYT** [comment]
> > I'm facing the same issue on my i9 9900k & 4070, is there no solution yet??
> 
> Nope, no fixes, this is just a visual issue tbh not that big of a deal.

It is a big issue when these dark spots are ALL over the DLC making a playthrough unpleasant.

**Comment by Missake212** [comment]
Compared to some other issues on Bloodborne this one is really small.

**Comment by diegolix29** [comment]
seems that some spots has been fixed kinda, im on intel i had the issue with the blaack stuff and some of them are working 

![Image](https://github.com/user-attachments/assets/5350526e-c9e2-447b-abd2-2bd94242065c)

![Image](https://github.com/user-attachments/assets/56d43822-691a-425b-ad74-023aaa1be12e)

![Image](https://github.com/user-attachments/assets/3dea10cb-0e53-447b-9448-c639fc710ae2)

tested on last main

**Comment by Missake212** [comment]
@diegolix29 still looks the exact same for me 

![Image](https://github.com/user-attachments/assets/1e01db88-30e6-4a22-9840-e68bfdb48dd0)

**Comment by Viscont23** [comment]
Having the same issue in the Old Hunters DLC. Doesn't look too bad in the Hunter's Nightmare and Fishing Hamlet (just a few areas are affected here and there), but in the Lady Maria's Astral Tower after beating Ludwig the issue goes really nuts: the whole area is insanely dark.

Here's how it looks on my PC:

![Image](https://github.com/user-attachments/assets/6b582c9a-de19-427c-94b6-4a2fbdc64e6b)

And this is the image from PS4 hardware (not my screenshot but I guess the difference is obvious):

![Image](https://github.com/user-attachments/assets/83d516dc-a8c2-4f2a-be16-1416c46d4f04)

Tested with "clean" game installation without any mods/patches.

My specs:
GeForce RTX 2080 8gb
Intel i7-7700
16gb RAM

**Comment by DaftRex211** [comment]
I have a question. I'm trying to get The Old Hunters DLC, but it doesn't appear in the Cheats or Patches section. Can anyone help me play it? My run was deleted.

**Comment by StevenMiller123** [comment]
The `Force Enable Old Hunters DLC` patch was removed, as it unlocks paid content for free.
You need to dump the DLC files from your PS4.

**Comment by carrrrrrrrot** [comment]
<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/3269b5aa-9f31-4303-9fbd-ab1459ac2174" />
<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/650e4f43-7da9-4f3f-a609-53a364a9fbf8" />

Changing this debug menu setting seems to fix the issue or at least bring it closer to how it should look

The setting is located in graphics->rend man->stage rend->sceneGUIWidget->GParamManager->Enable MultiThread

Tested on a xeon e5 1660v3 without any mods, aside from the one necessary to enable the debug menu itself.
